docs/advanced/peering/peering-via-cr.md

```diff
@@ -0,0 +1,428 @@
+
+## Declarative configuration of clusters authentication
+
+In this section we are going to configure the authentication between the clusters, allowing the cluster consumer to ask for resources.
+
+When authentication is manually configured, **the user is in charge of providing the credentials with the permission required** by the cluster consumer to operate.
+
+If your are not familiar with how authentication works in Kubernetes you can check [this documentation page](https://kubernetes.io/docs/reference/access-authn-authz/authentication/).
+You can also check [here to know how to issue a certificate for a user](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#normal-user).
+
+```{warning}
+Note that with EKS authentication via client certificate [is not directly supported](https://aws.amazon.com/blogs/containers/managing-access-to-amazon-elastic-kubernetes-service-clusters-with-x-509-certificates/).
+You can check [here](https://docs.aws.amazon.com/eks/latest/userguide/cluster-auth.html) how access control works in eks.
+```
+
+### Cluster consumer role binding (cluster provider)
+
+Once we created the credentials the consumer can work with, we will need to provide the minimum permission required by the consumer to operate.
+Note that **the cluster consumer will never directly create workloads on the remote cluster** and, at this stage, it should have only the permissions to create the liqo resources to ask for the approval of a `ResourceSlice`.
+
+To do so, we will need to bind the newly created user to the `liqo-remote-controlpane` role.
+This can be done **by creating the following `RoleBinding` resource in the tenant namespace of the cluster provider**:
+
+```{code} yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    liqo.io/remote-cluster-id: <CONSUMER_CLUSTER_ID>
+  name: liqo-binding-liqo-remote-controlplane
+  namespace: <TENANT_NAMESPACE>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: liqo-remote-controlplane
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: <USER_COMMON_NAME>
+```
+
+Where, when the user authenticate via certificate signed by the cluster CA, `USER_COMMON_NAME` is the CN field of the certificate.
+
+### Creation of a tenant for the cluster consumer (cluster provider)
+
+On the provider side, to allow the authentication of a consumer, we will need to create a `Tenant` resource for it.
+This resource is useful to control the remote consumer (e.g. if we the provider wants to prevent a remote consumer to negotiate more resources, it can set the tenant condition to `Cordoned`, stopping any other resources negotiation).
+
+Note that, in the case of declarative configuration, there won't be any handshake between the clusters, so we will need to configure the tenant so that it accepts the `ResourceSlice` of the given consumer, even though no handshake occurred.
```

Comment:
```suggestion
Note that, in the case of declarative configuration, there will not be any handshake between the clusters, so we will need to configure the tenant so that it accepts the `ResourceSlice` of the given consumer, even though no handshake occurred.
```